curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/646d404259f512c1a9e86ac1.md

```diff
@@ -7,11 +7,11 @@ dashedName: step-74
 
 # --description--
 
-In your `highPrecedence` function, declare a `regex` variable. Assign it a regular expression that matches a number (including decimal numbers) followed by a `*` or `/` operator followed by another number.
+In your `highPrecedence` function, you should use a regex to check if the provided string matches the pattern of a number followed by a `*` or `/` operator followed by another number. 
 
-Each number, and the operator, should be in separate capture groups.
+Your function should return a boolean value. Remember that you can use the `test()` method for this.
 
-Incorporate the regular expression you've defined into your `highPrecedence` function to test if the provided string `str` matches the pattern. Use the `test()` method on your `regex` variable and return the result.
+Each number, and the operator, should be in separate capture groups.
```

Comment:
Not sure why the capture groups are a requirement? We also do not test for it.

Reply:
Maybe they shouldn't be a hard requirement?

Reply:
Let's remove it as the groups are not needed.

If we did keep it, we should test for it, otherwise it doesn't make sense to have it in the requirements.

Reply:
Understood.